v2/emailpassword/serverless/with-aws-lambda/about.mdx

```diff
@@ -4,22 +4,23 @@ title: About
 hide_title: true
 ---
 
-<!-- COPY DOCS -->
+<!-- COPY SECTION -->
 <!-- ./thirdpartyemailpassword/serverless/with-aws-lambda/about.mdx -->
+<!-- 1 -->
 
 # About
 
 ## Overview steps
 
 1. Frontend setup
 2. Backend setup
-3. Creating an auth serverless function to expose SuperTokens APIs to the frontend
-4. AWS API Gateway setup
-5. Using SuperTokens functions in your APIs (for example for session verification).
-6. Next steps
+3. Using SuperTokens functions in your APIs (for example for session verification).
+4. Next steps
 
 ## [An example GitHub repo](https://github.com/supertokens/supertokens-auth-react/tree/master/examples/with-aws-lambda) 
```

Comment:
let's link to the node repo's example instead - we should remove the auth-react example and link to the node repo one in there too (since the node repo one actually runs and has tests for it, but the auth-react one doesn't, and don't want to maintain the same thing in two places)

Reply:
done